go.mod

```diff
@@ -0,0 +1,35 @@
+module github.com/arnaubennassar/rpc
```

Comment:
Should we adapt the naming accordingly? E.g. `github.com/0xPolygon/rpc`

Reply:
Probably haha